test/Transactions/Orleans.Transactions.Azure.Test/TransactionRecoveryTests.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Orleans.Transactions.AzureStorage.Tests
 {
-    [TestCategory("Azure"), TestCategory("Transactions"), TestCategory("Functional")]
+    [TestCategory("Azure"), TestCategory("Transactions")]
```

Comment:
Without the Functional category, these tests will never run

Reply:
This is buy design, at this point.  They are mostly stable now, but still have some odd failures i'm working out, and can run longer than we usually like.  Still working out kinks in the system :/